assignments/this.js

```diff
@@ -1,46 +1,43 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
+* 1.  2 things to descibe the principles of this and its as follows:
     In a method, this refers to the owner object.
-    Alone, this refers to the global object.
     In a function, this refers to the global object.
-    
-    In an event, this refers to the element that received the event.
-    Methods like call(), and apply() can refer this to any object.
 
-* 2. 
-* 3. 
-* 4. 
+* 2.  we use it when calling a function using "." or dot (this.function)
+
+* 3. launches with out a return function.
+* 4.  Methods like call(), and apply() can refer this to any object.
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
 // ===========code example for Window Binding========
 // method
-// Create an object:
+// Create an object:   when i created the guy in .....speak => 
 const person = {
-    firstName: "John",
-    lastName : "Doe",
+    firstName: "Tale",
+    lastName : "Anto",
     id     : 26,
     fullName : function() {
       return this.firstName + " " + this.lastName;
```

Comment:
`this` in this particular case will beind to the `person` so it is implicit bind.